readme.md

```diff
@@ -50,19 +50,20 @@ $ np --help
       patch | minor | major | prepatch | preminor | premajor | prerelease | 1.2.3
 
   Options
-    --any-branch        Allow publishing from any branch
-    --branch            Name of the release branch (default: master)
-    --no-cleanup        Skips cleanup of node_modules
-    --no-tests          Skips tests
-    --yolo              Skips cleanup and testing
-    --no-publish        Skips publishing
-    --preview           Show tasks without actually executing them
-    --tag               Publish under a given dist-tag
-    --no-yarn           Don't use Yarn
-    --contents          Subdirectory to publish
-    --no-release-draft  Skips opening a GitHub release draft
-    --test-script       Name of npm run script to run tests before publishing (default: test)
-    --no-2fa            Don't enable 2FA on new packages (not recommended)
+    --any-branch            Allow publishing from any branch
+    --branch                Name of the release branch (default: master)
+    --no-cleanup            Skips cleanup of node_modules
+    --no-tests              Skips tests
+    --yolo                  Skips cleanup and testing
+    --no-publish            Skips publishing
+    --preview               Show tasks without actually executing them
+    --tag                   Publish under a given dist-tag
+    --no-yarn               Don't use Yarn
+    --contents              Subdirectory to publish
+    --no-release-draft      Skips opening a GitHub release draft
+    --release-draft-only    Only opens a GitHub release draft for the latest published version
```

Comment:
This makes it sound that without the flag it publishes but opens the draft for the non-latest version or something. I think this could be written clearer.

Reply:
Yeah, you're right.

Do you prefer "Only opens a GitHub release draft"? It's more concise. 😁

Reply:
Yes